.gitignore

```diff
@@ -0,0 +1 @@
+/.vscode
```

Comment:
I would prefer these kind of ignores not be in the project, they should be in your global gitignore.